### ISSUE TITLE
Unable to read config.json

### ISSUE THREAD

**Issue by soullivaneuh** [maglnet/ComposerRequireChecker#56]
While trying to apply #37:

```
sullivan@b6636163567b:/code$ composer-require-checker check --config-file config.json --ignore-parse-errors -vvv
ComposerRequireChecker 0.2.0

In JsonLoader.php line 25:
                                                           
  [ComposerRequireChecker\Exception\NotReadableException]  
  unable to read config.json                               
                                                           

Exception trace:
 ComposerRequireChecker\JsonLoader->__construct() at phar:///usr/bin/composer-require-checker/src/ComposerRequireChecker/Cli/CheckCommand.php:122
 ComposerRequireChecker\Cli\CheckCommand->getCheckOptions() at phar:///usr/bin/composer-require-checker/src/ComposerRequireChecker/Cli/CheckCommand.php:64
 ComposerRequireChecker\Cli\CheckCommand->execute() at phar:///usr/bin/composer-require-checker/vendor/symfony/console/Command/Command.php:252
 Symfony\Component\Console\Command\Command->run() at phar:///usr/bin/composer-require-checker/vendor/symfony/console/Application.php:865
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/bin/composer-require-checker/vendor/symfony/console/Application.php:241
 Symfony\Component\Console\Application->doRun() at phar:///usr/bin/composer-require-checker/vendor/symfony/console/Application.php:143
 Symfony\Component\Console\Application->run() at phar:///usr/bin/composer-require-checker/bin/composer-require-checker.php:32
 include() at /usr/bin/composer-require-checker:10

check [--config-file CONFIG-FILE] [--ignore-parse-errors] [--] [<composer-json>]

sullivan@b6636163567b:/code$ ll config.json 
-rw-rw-r-- 1 sullivan sullivan 63 Feb 27 16:27 config.json
sullivan@b6636163567b:/code$ cat config.json 
{
  "symbol-whitelist" : [
    "Composer\\Script\\Event"
  ]
}
```

**Comment by cxj** [comment]
This appears to be a bug in handling relative path names to the config file.  If you use an absolute path (e.g. `/Users/foo/project/config.json`) it will work.  I am seeing this problem as well:
```
554:~/gits/webapp$ composer-require-checker check --config-file ./config.json
ComposerRequireChecker 2.0.0

In JsonLoader.php line 25:

  unable to read ./config.json


check [--config-file CONFIG-FILE] [--ignore-parse-errors] [--] [<composer-json>]

555:~/gits/webapp$ ls -l config.json
-rw-r--r--  1 user  staff  37 Sep 16 16:14 config.json
```

My SWAG is that this is because of the way the PHAR is constructed, and the current working directory is not what the CLI expects it to be.

**Comment by Ocramius** [comment]
Related? https://github.com/maglnet/ComposerRequireChecker/pull/141

**Comment by localheinz** [comment]
Any new on this one?

To be honest, it's a bit annoying - the recommended installation method is as PHAR, but specifying the configuration file does not work.

**Comment by localheinz** [comment]
Here's a good workaround:

```diff
-$ tools/composer-require-checker check --config-file=composer-require-checker.json
+$ tools/composer-require-checker check --config-file=$(pwd)/composer-require-checker.json
```

**Comment by fredden** [comment]
I think this has been resolved in https://github.com/maglnet/ComposerRequireChecker/pull/511, which landed in version 4.10.0.
